### DISCUSSION TITLE
Can one client accommodate options in multiple situations?

### DISCUSSION THREAD

**Discussion by Qyuee** [go-resty/resty#736] [Q&A] [answered]
Can one client accommodate options in multiple situations?

Only one resty client is created, and if I want to set the retryCount option differently when performing each request, how should I handle it?

**Comment by slessard** [comment, marked as answer]
This likely could be accomplished by adding a retry condition func to the request

```
	client := resty.New()
	req := client.R()
	req.AddRetryCondition(func(response *resty.Response, err error) bool {
		if response.Request.Attempt > 3 {
			return false
		}
		return true
	})
```
